mapsforge-map/src/main/java/org/mapsforge/map/rendertheme/rule/RenderThemeHandler.java

```diff
@@ -282,8 +284,8 @@ else if ("overlay".equals(qName)) {
                 byte minZoom = 9;
                 byte maxZoom = 17;
                 byte layer = 5;
-                short magnitude = 128;
-                boolean always = false;
+                int magnitude = Parameters.HILLSHADING_MAGNITUDE_DEFAULT;
```

Comment:
@Sublimis `RenderThemeHandler.HILLSHADING_MAGNITUDE` should also become `int`.

Reply:
It is an int, `public static final int HILLSHADING_MAGNITUDE_DEFAULT = 128;` ?

Reply:
@Sublimis I wrote about the existing in `RenderThemeHandler` to become an `int`:
```java
public static short HILLSHADING_MAGNITUDE = -1;
```